### PR TITLE
Setup Initial app layout

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/platform_constants.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+String get apiKey => _constants['apiKey'];
+String get authDomain => _constants['authDomain'];
+String get databaseURL => _constants['databaseURL'];
+String get projectId => _constants['projectId'];
+String get storageBucket => _constants['storageBucket'];
+String get messagingSenderId => _constants['messagingSenderId'];
+String get logUrl => _constants['logUrl'];
+String get publishUrl => _constants['publishUrl'];
+String get smsTopic => projectId + '-sms-channel-topic';
```
Comment (marianamarasoiu) [inline webapp/lib/platform_constants.dart]: I don't think we need the `smsTopic` constant here, it's nook specific

**Review by marianamarasoiu — APPROVED** [review]
LGTM!

File: webapp/lib/platform_constants.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/platform_constants.dart]: Removed

**Review by marianamarasoiu — APPROVED** [review]

**Comment by ktkization** [comment]
@marianamarasoiu Can you please merge. I don't have access to Lark repos

**Comment by marianamarasoiu** [comment]
yes, sorry, forgot about that, merging now!

**Pull request merged**